README.md

```
@@ -38,12 +54,24 @@ AI: *Reads relevant documents, creates analysis page*
-### External AI edits your workspace
+### Script or terminal, same workspace
 ```bash
-# Install MCP server
-npm install -g pagespace-mcp@latest
+$ pagespace tasks create <listId> --title "Ship the Q3 report" --priority high
+$ pagespace agents ask <agentId> "Summarize what changed on the roadmap this week"
+```
 
-# In Claude Desktop
+### An agent that can actually ship code
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)